### PR TITLE
Chore hidden results for choices

### PR DESCRIPTION
**Pull request by luchotc**
I'll add a visual demo for the changes later

### CODE REVIEW

File: app/helpers/icons_helper.rb [hunk 1]
```diff
@@ -19,25 +19,37 @@ def language_icon(language)
...
   end
 
   def discussion_status_fa_icon(discussion)
-    status_fa_icon(discussion.status)
+    contextualization_fa_icon(discussion)
   end
 
-  def icon_for_status(s)
-    iconized = s.iconize
+  def icon_for(iconizable)
```
Comment (luchotc) [inline app/helpers/icons_helper.rb]: Iconizable is not a word so we should change it. I hear suggestions :smile:.

File: lib/mumuki/laboratory/controllers/results_rendering.rb [hunk 2]
```diff
@@ -36,7 +36,6 @@ def progress_json
   end
 
   def render_results_html(results_partial, assignment)
-    results_partial = 'out_of_attempts' unless assignment.attempts_left?
```
Comment (luchotc) [inline lib/mumuki/laboratory/controllers/results_rendering.rb]: I'm delegating this one to the view now

File: app/views/layouts/_timer.html.erb [hunk 3]
```diff
@@ -4,7 +4,7 @@
     <i class="timer timer-text text-center"><%= t :time_left %></i>
     <i id="timer" class="timer timer-text text-center"></i>
     <script>
-      mumuki.startTimer('<%= end_time %>');
+      mumuki.startTimer('<%= end_time&.iso8601 %>');
```
Comment (luchotc) [inline app/views/layouts/_timer.html.erb]: This one fixes the broken timer on exams. 
 ActiveSupport::TimeWithZone (Time as well) doesn't output valid standard date formats in to_s by default

File: app/views/exercise_solutions/_contextualization_results_body.html.erb [hunk 4]
```diff
@@ -1,5 +1,5 @@
-<% if contextualization.results_visible? %>
-  <div>
+<div>
```
Comment (luchotc) [inline app/views/exercise_solutions/_contextualization_results_body.html.erb]: I changed the name of this partial as the title is always shown and I found it confusing

File: app/helpers/icons_helper.rb (hunk 1 above)
Comment (felipecalvo) [reply app/helpers/icons_helper.rb]: I think iconizable is the clearest choice.

File: app/helpers/icons_helper.rb [hunk 5]
```diff
@@ -1,7 +1,7 @@
 module IconsHelper
   #FIXME refactor names
```
Comment (felipecalvo) [inline app/helpers/icons_helper.rb]: Think you can delete this line? You _did_ refactor the names.

File: Gemfile [hunk 6]
```diff
@@ -28,5 +28,5 @@ group :development do
   gem 'web-console'
   gem 'codeclimate-test-reporter', require: nil
 end
-#
-gem 'mumuki-domain', path: '/home/lucho/mumuki/mumuki-domain'
```
Comment (felipecalvo) [inline Gemfile]: :laughing:

**Pull request merged**